src/components/options-list-wrapper/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ComponentProps, ReactElement, useRef } from 'react';
+import { Animated, StyleProp, View, ViewStyle } from 'react-native';
+
+import { ANIMATION_DURATION } from '../../constants/styles';
+import type { OptionalToRequired } from '../../helpers';
+import type { Select } from '../../index';
+import type { OptionsList } from '../options-list';
+
+type FromSelectProps = Pick<ComponentProps<typeof Select>, 'isAnimated'>;
+type FromOptionListProps = Pick<ComponentProps<typeof OptionsList>, 'isOpened'>;
```

Comment:
you can extract isAnimated here `type FromOptionListProps = Pick<ComponentProps<typeof OptionsList>, 'isOpened' | 'isAnimated'>;` because OptionList inherited isAnimated from props Select